data/json/items/obsolete.json

```diff
@@ -1896,5 +1896,40 @@
     "price": 450000,
     "weight": "100 g",
     "difficulty": 8
+  },
+  {
+    "id": "akmbigmag",
```

Comment:
Migrate these, don't stick them in obsolete. There's no need to have them hang around if you're going to provide replacements.